frontend/help.md

```diff
@@ -17,9 +17,32 @@ There are currently two kinds of webhooks supported for automatically building y
 
 You just put the URL into the webhook section of your repository configuration and your project will be built automatically.
 
-## Packaging assets into the zip-file
-
-If you want to add files from your repository to the zip-file you just have to create a file named `.artifact_files` and add each asset as an extra line to that file. For example if you have a file called `LICENSE` in the root of your repository and want this to get included into the build result you just add a line with the content `LICENSE` to your `.artifact_files`.
+## Using the `.gobuilder.yml` file
+
+To configure some aspects of your build you will need to create a `.gobuilder.yml` file in your repository root. This file currently has these options:
+
+- `readme_file`: The markdown file to display on the repository page in the web frontend. (Defaults to `README.md`)
+- `triggers`: A list of repositories to build after a successful build of your repository. This could be used to generate some CLI utilities sitting in subdirs of your repository.
+- `artifcats`: In this option you can list assets to include into the zip file created from the build. For example if you have a file called `LICENSE` in the root of your repository and want this to get included into the build result you just add a item with the content `LICENSE` to this array.
```

Comment:
Hmm the rename-map is currently out-of-scope as the rsync mechanism does not support renaming while synching… But please open a new feature request for this. I'll do some investigation how to do that sync without rsync to include renaming while syncing.